GOVERNANCE.md

```diff
@@ -23,6 +23,19 @@ Maintainers MUST remain active. If they are unresponsive for >3 months, they can
 
 If a maintainer violates our [Code of Conduct][code-of-conduct], all maintainers will discuss the incident allowing the maintainer to give background and other maintainers will cast a vote. If the vote results in a [super-majority][super-majority], then the maintainer needs to step down and leave the maintainer board.
 
+#### Approving & Merging PRs
+
+Maintainers will approve PRs only when:
+
+- It meets the [scaler requirements](https://github.com/kedacore/governance/blob/main/SCALERS.md)
+- It meets the [deprecation/breaking change requirements](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
+- It is in line with the goal of the project
+- It does not violate the [code of conduct](https://github.com/kedacore/governance/blob/main/CODE_OF_CONDUCT.md).
+
+In case of dispute, other maintainers are requested to review and agree what the best outcome is of a PR.
+
+Once a single maintainer approves a PR, they can merge it unless they want to have a second pair of eyes. (optional)
+
 ### Becoming A Maintainer
```

Comment:
```suggestion
Once a single maintainer approves a PR, they can merge it unless they want to have a second pair of eyes. (optional)

This rules apply to all the repos whose code will impact directly in the end-users, like [`keda`](https://github.com/kedacore/keda), [`charts`](https://github.com/kedacore/charts), [`keda-docs`](https://github.com/kedacore/keda-docs), etc.
Other repositories used for internal tooling, like [`testing-infrastucture`](https://github.com/kedacore/testing-infrastructure), [`test-tools`](https://github.com/kedacore/test-tools), etc, are excluded from these rules and maintainers can merge any commit without requiring any other approval than themself (unless they want to have a second pair of eyes)

Exceptionally and always due to good and important reasons, maintainers can merge changes without requiring additional approval, but they must notify the other maintainers about the action.

### Becoming A Maintainer
```

Reply:
I'm not sure if we need special rules, care to elaborate on this please?

Reply:
Sure, xD
some repos, such as test-tool, testing-infrastructure  are repos where changes don't affect to public artifacts but where there can be multiple changes at once if we are spinning up something (for example, I needed to merge some commits generating the K6 runner image, or when there were terraform issues with managed azure prometheus). If I had to ask for a review on any change, it'd have been terrible.

I totally agree with having a review on repos with public impact, but for internal stuff I think that requiring an approval is something that slows the process unnecessarily.

About the last paragraph, I see this governance as "a contract that we are signing", I prefer to keep the door open for exceptional cases. we can rephrase it as: "Exceptional situations could have exceptional responses, notifying always to other maintainers". But I won't close all the doors for rare situations.

Maybe the problem is that I understand this as a working way that we must follow at 100% of the cases and it's more a genera guidelines about how we work